### DISCUSSION TITLE
How to handle errors in side-channel streams?

### DISCUSSION THREAD

**Discussion by lostmsu** [microsoft/vs-streamjsonrpc#974] [Q&A]
I built a client-server pair that uses multiplexed streams, and when I interrupt the server the client simply sees the stream it got as ended with no exception thrown or any indication anything went wrong on the server side. How do I handle that?

**Comment by AArnott** [comment]
It sounds like you want this feature: https://github.com/dotnet/Nerdbank.Streams/issues/299, which first shipped in [Nerdbank.Streams 2.10.37-alpha](https://github.com/dotnet/Nerdbank.Streams/releases/tag/v2.10.37-alpha). Can you confirm which version of Nerdbank.Streams you're using?
